lxc/image.go

```diff
@@ -726,6 +726,7 @@ func (c *cmdImageImport) Run(cmd *cobra.Command, args []string) error {
 		Quiet:  c.global.flagQuiet,
 	}
 
+	imageType := "container"
```

Comment:
@stgraber this constant could be used here https://github.com/lxc/lxd/blob/master/shared/api/instance.go#L14

Reply:
See above